page_objects/electric_bikes.py

```diff
@@ -0,0 +1,81 @@
+from selenium.webdriver.common.by import By
+from utilities.ui_utilities.base_page import BasePage
+
+
+class ElectricBikesPage(BasePage):
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    __electric_bikes_title = (By.XPATH, '//h1[text()="Electric bikes"]')
+    __category_el_mountain_bikes = (By.XPATH, '//em[text()="Electric mountain bikes"]')
+    __category_el_hybrid_bikes = (By.XPATH, '//em[text()="Electric hybrid bikes"]')
+    __category_electra_e_bikes = (By.XPATH, '//em[text()="Electra e-bikes"]')
+    __category_electric_road_bikes = (By.XPATH, '//em[text()="Electric road bikes"]')
+    __el_mnt_bike_title = (By.XPATH, '//h1[text()="Electric mountain bikes"]')
+    __el_rd_bike_title = (By.XPATH, '//h1[text()="Electric road bikes"]')
+    __el_e_bike_title = (By.XPATH, '//h1[text()="GO! E-BIKES"]')
+    __el_hb_bike_title = (By.XPATH, '//h1[text()="Electric city bikes"]')
+    __first_bike_in_list = (
+        By.XPATH, '//div[@class="searchresultslistcomponent"]/ul/li[1]/div')
+    __close_geo_warning = (By.XPATH, '//i[@class="md-24"]')
+    __accept_cookie_button = (By.XPATH, '//a[@id="CybotCookiebotDialogBodyButtonAccept"]')
+    __biggest_bike_size = (By.XPATH, '//div[@class="mb-1"]/div/div/button[last()]')
+    __add_to_cart_button = (By.XPATH, '//button[@id="addToCartButton"]')
+
+    def geo_popup_check(self):
+        if self._is_displayed(self.__close_geo_warning):
+            self._click_via_js(self.__close_geo_warning)
+            return self
+        else:
+            return self
+
+    def accept_cookie(self):
+        if self._is_displayed(self.__accept_cookie_button):
+            self._click_via_js(self.__accept_cookie_button)
+            return self
+        else:
+            return self
+
+    def check_electric_bikes_page(self):
```

Comment:
we do not check anything on page opjects. All function names which return bool value should start from is_

please fix all around project